### ISSUE TITLE
Property 'ApiClient' does not exist on type 'typeof asana'. Did you mean 'Client'?ts(2551)

### ISSUE THREAD

**Issue by Strooss** [Asana/node-asana#296]
Types don't really match the code

**Comment by jv-asana** [comment]
Hi @Strooss which version of the node-asana are you using? In our v1.X.X (EX:[v1.0.2](https://www.npmjs.com/package/asana/v/1.0.2)) you would use `asana.Client` while in v3.X.X [EX: [v3.0.4](https://www.npmjs.com/package/asana/v/3.0.4)] it's `asana.ApiClient`

**Comment by oev-chrisbennett** [comment]
@Strooss the `@types/asana` package matches the types in the previous version. 

```
npm remove @types/node
rm -rf node_modules
npm install
```
then restart your editor, and the type issues should disappear. Sadly you'll have no types 😞

**Comment by Strooss** [comment]
Yes im using the last version (node-dsk-v3). and yes the package don't have types right?

**Comment by jv-asana** [comment]
Hi @Strooss, @oev-chrisbennett is correct.

The [@types/asana](https://www.npmjs.com/package/@types/asana) package only works with [node-asana v1](https://www.npmjs.com/package/asana/v/1.0.5) client library version.

[node-asana v3](https://www.npmjs.com/package/asana/v/3.0.7?activeTab=versions) client library version is not supported.

For context, we did not write or maintain the [@types/asana](https://www.npmjs.com/package/@types/asana) package. It looks like a external dev worked on this and had to manually maintain it by updating it when our API changes.

Taking a look at [the commit history](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/master/types/asana/index.d.ts) for [@types/asana](https://www.npmjs.com/package/@types/asana) it looks like the last major update to the types was 2 years ago.

**Comment by Xanacas** [comment]
I see, that you have a lot of jsdocs in the source, which should enable typescript to create types while using this lib. 
However, it's not working for me. any ideas?

**Comment by dobobaie** [comment]
Guys, your v1 was way better than the v3... such a disappointment for the DX... :/
